fuzz/fuzz_targets/jit_cranelift.rs

```diff
@@ -48,14 +48,7 @@ fuzz_target!(|module: WasmSmithModule| {
         Ok(_) => {}
         Err(e) => {
             let error_message = format!("{}", e);
-            if error_message
-                .contains("RuntimeError: memory out of bounds: data segment does not fit")
-                || error_message
-                    .contains("RuntimeError: table out of bounds: elements segment does not fit")
-                || error_message.contains(
-                    "RuntimeError: out of bounds table access: elements segment does not fit",
-                )
-            {
+            if error_message.contains("RuntimeError: ") && error_message.contains("out of bounds") {
```

Comment:
Maybe we could use `error.message.starts_with` rather than `.contains` for `"RuntimeError: "` (https://doc.rust-lang.org/std/primitive.str.html#method.starts_with), thoughts?

Reply:
Not a big deal though.

Reply:
https://github.com/wasmerio/wasmer/pull/2301